extensions/typescript-language-features/src/tsServer/serverProcess.electron.ts

```diff
@@ -278,7 +278,7 @@ export class ElectronServiceProcessFactory implements TsServerProcessFactory {
 		}
 
 		const childProcess = execPath ?
-			child_process.spawn(execPath, [...execArgv, tsServerPath, ...runtimeArgs], {
+			child_process.spawn(`"${execPath}"`, [...execArgv, tsServerPath, ...runtimeArgs], {
```

Comment:
I think you should instead do `JSON.stringify(execPath)` which will handle quotes in the path as well.

Reply:
Nice catch! Updated and tested again on Windows/Linux.